### PR TITLE
azure - update resource examples in docs

### PR DESCRIPTION
**Pull request by erwelch**
Addresses: #3868 

* Removes examples irrelevant to the resource
* Adds some more relevant examples to some of the resources with no other examples
* Removes notify from all the resource examples 

Example rendered docs: https://erwelchsqtest.blob.core.windows.net/cc-docs/html/index.html

### CODE REVIEW

**Comment by stefangordon** [comment]
Oh yeah this is waaaay better.   Thank you.

File: docs/source/azure/policy/resources/keyvault.rst [hunk 1]
```diff
@@ -39,38 +39,12 @@ Actions
 Example Policies
 ----------------
 
-This set of policies will mark all Key Vaults for deletion in 7 days that have 'test' in name (ignore case),
-and then perform the delete operation on those ready for deletion.
+This policy will find all KeyVaults with 10 or less API Hits over the last 72 hours
```
Comment (aluong) [inline docs/source/azure/policy/resources/keyvault.rst]: I think there are some nice examples around keys and permissions we can link here.

Comment (erwelch) [reply docs/source/azure/policy/resources/keyvault.rst]: There are other examples already! I just deleted the one that deletes in 7 days.

File: docs/source/azure/policy/resources/publicip.rst [hunk 2]
```diff
@@ -19,38 +19,12 @@ Actions
 Example Policies
 ----------------
 
-This set of policies will mark all public IP addresses for deletion in 7 days that have 'test' in name (ignore case),
-and then perform the delete operation on those ready for deletion.
+This policy will find all public IP addresses under DDoS attack over the last 72 hours
```
Comment (aluong) [inline docs/source/azure/policy/resources/publicip.rst]: I don't quite understand this example. Can we provide more details on why this indicates a DDoS attack?

Comment (erwelch) [reply docs/source/azure/policy/resources/publicip.rst]: Github is showing the diff weird. If you expand the 3 lines it collapsed you'll see this was a previous example and the metric it 'IfUnderDDosAttack'

File: docs/source/azure/policy/resources/publicip.rst [hunk 3]
```diff
@@ -19,38 +19,12 @@ Actions
 Example Policies
```
Comment (aluong) [inline docs/source/azure/policy/resources/publicip.rst]: Maybe we can add the example of finding VMs with public IPs, because that is a common policy.

Comment (erwelch) [reply docs/source/azure/policy/resources/publicip.rst]: That is under the VM examples!

**Pull request merged**